packages/ng/forms/phone-number-input/phone-number-input.component.ts

```diff
@@ -42,7 +65,7 @@ const PREFIX_ENTRIES = getCountries().map((country) => ({
 export class PhoneNumberInputComponent implements ControlValueAccessor, Validator {
 	@Input() label: string;
 
-	#onChange?: (value: E164Number | ValidatePhoneNumberLengthResult) => void;
+	#onChange?: (value: E164Number) => void;
```

Comment:
The reason why we had `ValidatePhoneNumberLengthResult` was for the internal validator to be able to tell when a number is invalid and set itself to error state without having to specify some kind of `PhoneNumberFormatValidator`

Reply:
I have some doubts about this.
Is it a good practice to use `registerOnChange`/`onChange` for validation ?
I thought it was for control's value only, which is here the phone number